src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -1321,6 +1321,25 @@ typedef struct {
     RtcStatsObject rtcStatsObject; //!< Object that is populated by the SDK on request
 } RtcStats, *PRtcStats;
 
+typedef struct {
```

Comment:
I have multiple issues with this approach.

1) Most importantly, why are we carrying the extra params needed for the stats in the frame itself? Do we assume the encoder will be changing the bitrate and the resolution with each frame?
2) The backward compat will only work if we modify the version of the underlying frame struct to some sentinel version that will allow the code to access the bits following it.

How else can the encoder communicate the necessary stats to the SDK? Perhaps a new API? In a real-life situation, are there any encoders that actually expose the bitrate and the frame encoder time? I haven't seen any. 

The width and height can be extracted from the codec private data - namely, from SPS for H264.

Reply:
> encoder will be changing the bitrate and the resolution with each frame?
changing target bitrate and even resolutiin is a part of the spec

Reply:
i dont like this approach either
but couldnt come up with anything better
ideas?
separate API?

Reply:
Certainly, a separate API is a lot better than overloading the frame - something like onEncoderStatsChanged or similar. What I am not certain about is how this will be wired in but I assume some platforms might allow getting these stats from the encoder.

Reply:
`onEncoderStatsChanged` makes sense